usaspending_api/search/tests/test_spending_by_award_type_count.py

```diff
@@ -30,6 +74,7 @@ def test_spending_by_award_type_success(client, refresh_matviews):
     assert resp.status_code == status.HTTP_200_OK
 
 
+@pytest.mark.skip
```

Comment:
Are these and subsequent skips something we think we can rectify (come up with a way to test the async code)? 
Problem identified and concept for a solution formed? 
Or is it looking like we may need to remove the tests?

Reply:
I haven't found a way yet to allow the async DB calls to run within the current pytest fixtures.

I'm sure there is a way to make new fixtures with new DB connections, but I have't found a good solution yet

Reply:
@tony-sappe this article gives a good synopsis of the problem and 2 reasonable solutions:
https://medium.com/ideas-at-igenius/testing-asyncio-python-code-with-pytest-a2f3628f82bc

> So, we need a way to tell pytest to use an event loop for executing tests defined as asyncio coroutines. 

Ultimately the recommendation is add in use of [`pytest-asyncio`](https://github.com/pytest-dev/pytest-asyncio) (which the article builds up to helpfully)

Reply:
I'll read the medium post. *However* a quick glance makes me think it doesn't describe the issue here.

What is occurring are the pytest fixtures are running using one connection, and this asyncio function is creating a new connection using a different library. What happens is the pytest fixture holds an idle transaction and locks the new async connection from reading the DB models.

To resolve this issue, I have two potential ideas:
1. Create new fixtures to setup/teardown using the new `asyncpg` library for only these tests in question
2. Find a way to exchange the db connection used by the fixtures